notebooks_tsqr/NightLog.ipynb

```diff
@@ -19,7 +19,8 @@
     "# Times Square replaces this cell with the user's parameters.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.    CONSDB_URL = "http://consdb-pq.consdb:8080/consdb" # Don't use this one
</pre><p>Is this always the same for all environments (tucson-teststand, summit, usdf)?  If not, it should be set based upon the EXTERNAL_INSTANCE_URL env var.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Yes, this is the same for both summit and usdf. I also have the note 'don't use this' because some updates need to be made to ConsDbClient to accept headers towards authenticating in a copacetic way according to sqre. 
Reference to consdb usage here: https://rubinobs.atlassian.net/wiki/spaces/~ktl/pages/55377993/ConsDB+Usage#ConsDBUsage-ConnectionInformation